sdk/anomalydetector/azure-ai-anomalydetector/src/samples/java/com/azure/ai/anomalydetector/DetectChangePoints.java

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.anomalydetector;
+
+import com.azure.ai.anomalydetector.models.ChangePointDetectRequest;
+import com.azure.ai.anomalydetector.models.ChangePointDetectResponse;
+import com.azure.ai.anomalydetector.models.TimeGranularity;
+import com.azure.ai.anomalydetector.models.TimeSeriesPoint;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.http.*;
+import com.azure.core.http.policy.AddHeadersPolicy;
+import com.azure.core.http.policy.AzureKeyCredentialPolicy;
+import com.azure.core.http.policy.HttpPipelinePolicy;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DetectChangePoints {
+
+    public static void main(final String[] args) throws IOException {
+        String endpoint = "<anomaly-detector-resource-endpoint>";
+        String key = "<anomaly-detector-resource-key>";
+        HttpHeaders headers = new HttpHeaders()
+            .put("Accept", ContentType.APPLICATION_JSON);
+
+        HttpPipelinePolicy authPolicy = new AzureKeyCredentialPolicy("Ocp-Apim-Subscription-Key",
+            new AzureKeyCredential(key));
+        AddHeadersPolicy addHeadersPolicy = new AddHeadersPolicy(headers);
+
+        HttpPipeline httpPipeline = new HttpPipelineBuilder().httpClient(HttpClient.createDefault())
+            .policies(authPolicy, addHeadersPolicy).build();
+        AnomalyDetectorClient anomalyDetectorClient = new AnomalyDetectorClientBuilder()
+            .pipeline(httpPipeline)
+            .endpoint(endpoint)
+            .buildClient();
+
+        Path path = Paths.get("./src/samples/java/sample_data/request-data.csv");
+        List<String> requestData = Files.readAllLines(path);
+        List<TimeSeriesPoint> series = requestData.stream()
```

Comment:
It would be good to add comments to describe what is happening in the samples to help the users follow the sample better.